Modules/Core/TestKernel/include/itkTestDriverIncludeRequiredIOFactories.h

```diff
@@ -15,20 +15,12 @@
  *  limitations under the License.
  *
  *=========================================================================*/
-#ifndef itkTestDriverIncludeRequiredIOFactories_h
-#define itkTestDriverIncludeRequiredIOFactories_h
 
+// Legacy file for backwards compatibility
 
-#include "itkTestDriverInclude.h"
-
-#ifdef __EMSCRIPTEN__
-#  include <emscripten.h>
-#endif
-
-void
-RegisterRequiredFactories();
+#ifndef itkTestDriverIncludeRequiredIOFactories_h
```

Comment:
I don't think that include guards are needed, as the included file will have them.

Reply:
I thought the include guards were part of the style guidelines? I'll try without them

Reply:
Include guards are needed to [prevent errors](https://www.geeksforgeeks.org/include-guards-in-c/), ITK's style guide just defines their style 😄

Reply:
Removing the include guards fails the KWStyle commit check:

```
Modules/Core/TestKernel/include/itkTestDriverIncludeRequiredIOFactories.h:17: error: #ifndef not defined
pre-commit hook failure
-----------------------

KWStyle check failed.

Line numbers in the errors shown refer to the file:
Modules/Core/TestKernel/include/itkTestDriverIncludeRequiredIOFactories.h.kws
```

Reply:
But maybe KWStyle might complain due to their lack. If it does, put them back in 😄

Reply:
> Removing the include guards fails the KWStyle commit check

Then keep them. I wrote the above comment while you were writing yours 😄

Reply:
Right, they're redundant here but not particularly messy. Will leave them in to make KWStyle happy.